luarules/gadgets/game_no_rush_mode.lua

```diff
@@ -73,16 +73,20 @@ if gadgetHandler:IsSyncedCode() then
 		if frame < norushtimer and (not TeamIDsToExclude[unitTeam]) then
 			local _, _, _, _, _, allyTeamID = Spring.GetTeamInfo(unitTeam)
 			if cmdID == CMD.INSERT then
+			  -- CMD.INSERT wraps another command, so we need to extract it
```

Comment:
Many gadgets will want to unwrap an INSERTed command, so perhaps it's worth making this a general `Spring.Utilities.` helper. Don't apply it across the board though because an inserted command is not always equivalent to a direct one and because smuggling a raw command is sometimes useful.